.gitlab-ci/packet.yml

```diff
@@ -73,6 +73,20 @@ packet_cleanup_old:
     - make cleanup-packet
   after_script: []
 
+packet_centos7-calico-ha-once-localhost:
+  stage: test
+  extends: .packet_pr
+  variables:
+    # This will instruct Docker not to start over TLS.
+    DOCKER_TLS_CERTDIR: ""
+    DOCKER_HOST: "tcp://docker:2375"
+  services:
+    - docker:26.1.4-dind
+
+packet_centos7-flannel-addons-ha:
+  stage: test
```

Comment:
remove this stage

Reply:
allow_failure:  we don't want to deal with centos failing right away, but still want to know when it happens